steps/utils.ts

```diff
@@ -1,3 +1,19 @@
+import slugify from 'slugify'
+import { cats } from 'lib/const'
+
+const formatcsv = (inputs: string): string[] => {
+  return inputs.split(',').map((input) => input.trim())
+}
+
 export const formatLanguages = (langs: string): string[] => {
-  return langs.split(',').map((lang) => lang.trim())
+  return formatcsv(langs)
+}
+
+export const formatSubjects = (subjects: string): string[] => {
+  return formatcsv(subjects).map((subject) => slugify(subject, { lower: true }))
+}
+
+export const verifySubjects = (subjects: string): boolean => {
```

Comment:
I don't get why we lookup indexes to verify subject validity, wouldn't it be enough to check that all subjects are among categories?